bin/plugin/commands/performance.js

```diff
@@ -225,6 +210,7 @@ async function runTestSuite( testSuite, performanceTestDirectory, runKey ) {
 async function runPerformanceTests( branches, options ) {
 	const runningInCI = !! process.env.CI || !! options.ci;
 	const TEST_ROUNDS = options.rounds || 1;
+	const artifactsPath = process.env.WP_ARTIFACTS_PATH || '';
```

Comment:
to double-check, if this is empty, it will print to the current directory? or should we pass `'./'` as the default? or something like `__dirName`?

Reply:
If this is empty, it will be resolved down the stream (per job) when the puppeteer env is set up (when running `npm run test:performance`): 
https://github.com/WordPress/gutenberg/blob/b3ab0eed6fcd921364f4ceda36ffb55cecf94c33/packages/scripts/config/jest-environment-puppeteer/index.js#L45-L49

Note that the `GITHUB_WORKSPACE` var is not available here when running in CI (as mentioned [here](https://github.com/WordPress/gutenberg/pull/48684#discussion_r1133783175)) so it will always resolve to the `cwd`. This was causing the artifacts to be saved along the `env` folders that we [create during the perf tests setup](https://github.com/WordPress/gutenberg/blob/4538099f41f81783720bc9c6d619bf06002ce79e/bin/plugin/commands/performance.js#L284) and the necessity to copy them later on to the workspace's `./__test-results` folder, so that they're available for the CI. Does that make sense?

Reply:
> Note that the `GITHUB_WORKSPACE` var is not available here when running in CI (...)

⚠️ This is (currently) true only for the performance tests because they're run via child processes. For the E2E tests that run directly from the CI, the parent process env variables are accessible here. I've made some changes in https://github.com/WordPress/gutenberg/pull/48684/commits/a53065b9b9dd37366df0eb4c5dfd3d0ee55ade22 to hopefully streamline this bit.